src/websocket/Socket.ts

```diff
@@ -3,7 +3,7 @@ import { WebSocket } from "ws";
 import { Command, Tetris } from "../core/Tetris.js";
 
 const DEFAULT_WS_CLOSURE = 1000;
-const DEFAULT_HEARTBEAT_INTERVAL = 30000;
+const DEFAULT_HEARTBEAT_INTERVAL = 5000;
```

Comment:
Same thing with https://github.com/NunkLabs/tetribass-client/pull/620. I suggest we add units here for constants